### PR TITLE
chore: upgrade cypress@12.17.2

### LINKED ISSUE #1522: [bug] - e2e tests fail with firefox
**Describe the bug**
As noted in #1521, firefox has been given quite a lot of flaky and failed tests. Ideally should be investigated further in the future (probably after a few more core updates are in to things like cypress and firebase)

### PR DESCRIPTION
**Pull request by benfurber**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Developer experience (improves developer workflows for contributing to the project)

## Description

Locally I'm finding no problem with the firefox tests, running on ci to see if there's issues.

## Git Issues

Closes #1522

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -172,6 +172,7 @@
     "concurrently": "^6.2.0",
     "cross-env": "^7.0.3",
     "cspell": "^6.1.0",
+    "cypress": "12.17.2",
```
Comment (thisislawatts) [inline package.json:175]: **issue (blocking):** It seems like there may have been a small oversight here. I've noticed that the `cypress` package has been added to the root application. We should avoid introducing this dependency in the root package.json file. Would you mind looking into this?

**Review by thisislawatts — CHANGES_REQUESTED** [review]
**praise:** Thanks for picking this item up @benfurber. One small fix and then it looks ready to merge.

File: package.json (hunk 1 above)
Comment (benfurber) [reply package.json]: Yes! Sloppy of me, apologies.

**Comment by benfurber** [comment]
@iSCJT - I forgot to ask earlier. Did you mean to add to this branch?

**Comment by benfurber** [comment]
@evakill Rebased your change in.

**Review by evakill — APPROVED** [review]

**Pull request merged**